sdk/analytics/src/main/java/com/klaviyo/analytics/networking/requests/PushTokenApiRequest.kt

```diff
@@ -26,6 +27,11 @@ internal class PushTokenApiRequest(token: String, profile: Profile) : KlaviyoApi
         const val ANDROID_TOKEN = "\$android_tokens"
     }
 
+    override var headers: Map<String, String> = mapOf(
+        HEADER_ACCEPT to TYPE_HTML,
```

Comment:
curious why we need to accept html?

Reply:
interesting....cant wait to get off this api

Reply:
It beats base64 in a GET url, but... only barely